src/Support/StormpathLaravelServiceProvider.php

```diff
@@ -109,8 +109,8 @@ private function warmResources()
                 'href' => $mapping->accountStore->href,
                 'name' => $mapping->accountStore->name,
                 'provider' => [
-                    'href' => $mapping->accountStore->provider->href,
-                    'providerId' => $mapping->accountStore->provider->providerId,
+                  'href' => isset($mapping->accountStore->provider->href) ? $mapping->accountStore->provider->href : null,
+                  'providerId' => isset($mapping->accountStore->provider->providerId) ? $mapping->accountStore->provider->providerId : null,
```

Comment:
I am actually wrong... Provider only lives on directories which means that provider would not exist.  What should be done here is set the acountStoreArray and check to see if the `$mapping->accountStore->provider`.  if it does, then we attach the provider array to it.  if not, provider array should not be included